pkgs/development/libraries/docopt_cpp/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, cmake, python }:
+
+stdenv.mkDerivation rec {
+  version = "0.6.2";
+  name = "docopt_cpp-${version}";
+
+  src = fetchFromGitHub {
+    owner = "docopt";
+    repo = "docopt.cpp";
+    rev = "v${version}";
+    sha256 = "1rgkc8nsc2zz2lkyai0y68vrd6i6kbq63hm3vdza7ab6ghq0n1dd";
+  };
+
+  buildInputs = [ cmake python ];
+
+  cmakeFlags = ["-DWITH_TESTS=ON"];
+
+  doCheck = true;
+
+  checkPhase = "python ./run_tests";
+
+  meta = with stdenv.lib; {
```

Comment:
no need of writing stdenv.lib in license and platforms when you do meta = with stdenv.lib